### PR TITLE
Add Next.js SSR example

### PR DESCRIPTION
**Pull request by sergiodxa**

### CODE REVIEW

File: next.js-ssr/README.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+```bash
+yarn init --yes
+```
+
+## Step 2: Install dependencies
+
+Add the project dependencies:
+
+```bash
+yarn add next react react-app
```
Comment (morajabi) [inline next.js-ssr/README.md]: Should this be `react-dom` instead of `react-app`?

Comment (sergiodxa) [reply next.js-ssr/README.md]: You are right, it was a typo, is fixed now, thanks!

File: README.md [hunk 2]
```diff
@@ -37,6 +37,7 @@ These deployments:
 | [misc-socket.io-chat](https://misc-socketio-chat-v2.now.sh) | [Node.js](https://en.wikipedia.org/wiki/Node.js) | The socket.io [chat demo](https://socket.io/demos/chat/) | 20.2M |
 | [misc-socket.io-whiteboard](https://misc-socketio-whiteboard-v2.now.sh) | [Node.js](https://en.wikipedia.org/wiki/Node.js) | The socket.io [whiteboard demo](https://socket.io/demos/whiteboard/) | 20.2M |
 | [misc-web-tty](https://misc-web-tty-v2.now.sh) | [Go](https://en.wikipedia.org/wiki/Go_(programming_language)) | A fully functional terminal in the browser powered by [gotty](https://github.com/yudai/gotty) | 6.1M |
+| [next.js-ssr](nextjs-ssr.now.sh) | [Node.js](https://en.wikipedia.org/wiki/Node.js) | A simple Next.js application with two pages and SSR enabled | 86.1M |
```
Comment (pranaygp) [inline README.md]: I think
> A simple Next.js application with Server Side Rendering
is enough

Comment (sergiodxa) [reply README.md]: Done

**Pull request merged**